### ISSUE TITLE
Please update to Ebean 11.17.x

### ISSUE THREAD

**Issue by PromanSEW** [playframework/play-ebean#155]
https://github.com/ebean-orm/ebean/compare/ebean-11.15.4...ebean-11.17.3

**Comment by KoenDG** [comment]
I just got this in my mail: https://groups.google.com/forum/#!topic/ebean/ppj7eGhN01s

The following functionality was added:

https://github.com/ebean-orm/ebean/issues/1407
https://github.com/ebean-orm/ebean/issues/1406
https://github.com/ebean-orm/ebean/issues/1405

That is some seriously useful stuff when you're using a few custom queries with big returns. Right now you have to receive a LinkedHashMap, which is big in memory. This should trim memory usage down a bit, by mere virtue of not bringing that LinkedHashMap into your scope and passing it around.

Those will be in 11.17.2, which isn't out yet.

**Comment by PromanSEW** [comment]
@KoenDG 1405 and 1406 are awesome features!
Like it so much.
Thanks for linking them.

**Comment by PromanSEW** [comment]
@KoenDG I see that I should override Ebean dependencies manually...
I think it will work fine without other changes in plugin's code.

**Comment by PromanSEW** [comment]
@KoenDG I have already tested 11.17.3, all works fine 👍
```scala
val jacksonVersion = "2.9.5"
dependencyOverrides ++= Seq(
  // ---------------- Jackson (remove after Play 2.7 release) -----------------
  "com.fasterxml.jackson.core" % "jackson-annotations" % jacksonVersion,
  "com.fasterxml.jackson.core" % "jackson-core" % jacksonVersion,
  "com.fasterxml.jackson.core" % "jackson-databind" % jacksonVersion,
  "com.fasterxml.jackson.datatype" % "jackson-datatype-jdk8" % jacksonVersion,
  "com.fasterxml.jackson.datatype" % "jackson-datatype-jsr310" % jacksonVersion,
  // ---------------------------------------------------------------------------
  "io.ebean" % "ebean" % "11.17.3",
  "io.ebean" % "ebean-agent" % "11.11.1"
)
```
You should also override Jackson because Ebean uses `2.9.5` itself: https://github.com/ebean-orm/ebean/issues/1376
Play `2.6.x` uses `2.8.x` branch now, `2.9.x` will be used in `2.7.x` only:
https://github.com/playframework/playframework/blob/773e372279630a4cd318c11a386ff6ed33028731/framework/project/Dependencies.scala#L28-L38
https://github.com/playframework/playframework/blob/master/framework/project/Dependencies.scala#L37-L44

**Comment by KoenDG** [comment]
@PromanSEW Nice bit of documentation there, thanks

**Comment by rbygrave** [comment]
> Ebean uses 2.9.5 (of Jackson) itself

Just note that Ebean doesn't strictly **_need_** Jackson 2.9.5.  You can use 2.8.x of Jackson with Ebean 11.17.x if you like - there is no strict requirement to upgrade the Jackson version.

**Comment by PromanSEW** [comment]
@KoenDG I have checked and updated my post.

**Comment by PromanSEW** [comment]
Fixed by #157
